### DISCUSSION TITLE
API Key Question

### DISCUSSION THREAD

**Discussion by Howie0928** [FlowiseAI/Flowise#3132] [General]
![457093562_1809590292903066_6106753593586796416_n](https://github.com/user-attachments/assets/5b1b9e85-737f-4e40-842e-9d70b46d5219)
![457585174_1868431420334024_1309286908115794461_n](https://github.com/user-attachments/assets/44ed2be5-91b5-4c11-a9dd-9bcb72370b09)

Hello everyone,

I'd like to ask why my Flowise is showing the message "You exceeded your current quota" even though my OpenAI API account is in a paid status and still has a $10 credit available. Does anyone know what might be causing this issue?

**Comment by ShermanKasaval** [comment]
Same issue here.
